solang-parser/Cargo.toml

```diff
@@ -18,6 +18,7 @@ lalrpop-util = "0.19"
 phf = { version = "0.10", features = ["macros"] }
 unicode-xid = "0.2.0"
 itertools = "0.10"
+serde = {version = "1.0.145", features = ["derive"]}
```

Comment:
This is not correctly formatted

Reply:
Hey @xermicus, 
I am currently building a tool for analyzing smart contracts (Security) => https://github.com/Ethnical/Swek3 
I would like to add this PR to convert AST to JSON and then use `ethers-solc` in the future also to integrate it `solang-parser`. :) 
Anyway, what do you mean by bad formatting for the `Cargo.toml` ? I guess you are speaking about some spacing formatting maybe?

Reply:
Thanks for sharing! So you are using it for static analysis?

>  I guess you are speaking about some spacing formatting maybe?
Yes sorry I wasn't super clear about that. See also the example [here](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/cargo.md)

Reply:
@xermicus Yes actually really nice to learn rust :)